packages/api/cms-api/src/blocks/createSeoBlock.ts

```diff
@@ -18,6 +18,7 @@ import {
     SimpleBlockInputInterface,
     TraversableTransformResponse,
 } from "@comet/blocks-api";
+import { WarningSeverity } from "@comet/blocks-api/lib/blocks/block";
```

Comment:
This should be part of the public API if it's used in other packages.

Reply:
done in: b1cdee46ef6c26fce88f08d0e0a5f1f9d6e4e943